app/views/sessions/authenticate.html.erb

```diff
@@ -1,74 +1,46 @@
-
 <% login_info = get_login_info %>
 
-<h3>
-  Hi<%= login_info[:names].size == 1 ? " #{login_info[:names].first}!" : "!" %>
-</h3>
+<div class="ox-card login">
 
-<div id="options" class="row">
-  <div class="col-sm-12">
+  <h1 class="title"><%= t '.name_greeting', name: login_info[:names].first %></h1>
 
-    <% if login_info[:providers].include?('identity') %>
-      <div class="button-form-group">
-        <%= form_tag '/auth/identity/callback' do %>
-          <input type='hidden' id='auth_key' name='auth_key' class="form-control"
-                 value='<%= login_info[:username_or_email] %>'/>
-          <div class="form-group">
-            <label for='password'><%= t :".password" %></label>
-            <input type='password' id='password' name='password' class="form-control"/>
-          </div>
+  <% if login_info[:providers].include?('google_oauth2') %>
+    <%= link_to "/auth/google_oauth2", class: 'btn btn-social btn-google',
+                                       id: 'google-login-button' do %>
+      <span class="fa fa-google"></span> <%= t :".sign_in_with_google" %>
+    <% end %>
+  <% end %>
 
-          <div class='password-actions'>
-            <button type="submit" class="btn btn-primary"><%= t :".sign_in" %></button>
-          </div>
+  <% if login_info[:providers].include?('facebook') %>
+    <%= link_to "/auth/facebook", class: 'btn btn-block btn-social btn-facebook',
+                                  id: 'facebook-login-button' do %>
+      <span class="fa fa-facebook"></span> <%= t :".sign_in_with_facebook" %>
+    <% end %>
+  <% end %>
 
-        <% end %>
-      </div>
+  <% if login_info[:providers].include?('identity') %>
+    <% if login_info[:providers].many? %>
 
-      <% if login_info[:providers].size > 1 %>
-        <div id="horizontal-or-bar">
-          <span><%= t :".separator_or" %></span>
-        </div>
-      <% end %>
-    <% end %>
+    <div class="separator-line" data-text="">
+      <%= t '.separator_or' %>
+    </div>
 
-    <% if login_info[:providers].include?('facebook') %>
-      <%= link_to "/auth/facebook", class: 'btn btn-block btn-social btn-facebook',
-                                    id: 'facebook-login-button' do %>
-        <span class="fa fa-facebook"></span> <%= t :".sign_in_with_facebook" %>
-      <% end %>
     <% end %>
+    <%= lev_form_for :login, url: '/auth/identity/callback' do |f| %>
 
-    <% if login_info[:providers].include?('google_oauth2') %>
-      <%= link_to "/auth/google_oauth2", class: 'btn btn-block btn-social btn-google',
-                                         id: 'google-login-button' do %>
-        <span class="fa fa-google"></span> <%= t :".sign_in_with_google" %>
-      <% end %>
-    <% end %>
+      <%= f.password_field :password, placeholder: t(".password"), class: 'wide'  %>
 
-  </div>
-</div>
+      <%= f.hidden_field :username_or_email, placeholder: t(".password"),
```

Comment:
Yep, also other attributes that don't make sense hidden fields, like class and autofocus.  (as you can tell I copied the `text_field` for that 😁